packages/mdc-ripple/_mixins.scss

```diff
@@ -169,6 +168,41 @@ $mdc-ripple-common-styles-emitted_: false !default;
   @include mdc-states-hover-opacity(map-get($opacity-map, "hover"));
   @include mdc-states-focus-opacity(map-get($opacity-map, "focus"), $has-nested-focusable-element);
   @include mdc-states-press-opacity(map-get($opacity-map, "press"));
+
+  @if $activated-color {
+    &--activated {
```

Comment:
does --activated work with all existing components? Or will this be a new modifier for some components?